homeassistant/components/amcrest/__init__.py

```diff
@@ -201,11 +201,15 @@ def _monitor_events(hass, name, api, event_codes):
     while True:
         api.available_flag.wait()
         try:
-            for code, start in api.event_actions("All", retries=5):
-                event_data = {"camera": name, "event": code, "payload": start}
+            for code, payload in api.event_actions("All", retries=5):
+                event_data = {"camera": name, "event": code, "payload": payload}
                 hass.bus.fire("amcrest", event_data)
                 if code in event_codes:
                     signal = service_signal(SERVICE_EVENT, name, code)
+                    start = any(
+                        str(key).lower() == "action" and str(val).lower() == "start"
```

Comment:
Why the casting to lowercase? I would not assume an API to return random casing of keys and values. Is there more than one expected value?